cal/cal/static/js/services.js

```diff
@@ -0,0 +1,131 @@
+/*jslint devel: true */
+/*jslint browser: true */
+/*jslint jquery: true */
+/*global getCookie */
```

Comment:
Any way to disable this feature of linter? Seems like way too many excess comments. I guess it's fine but seems like there's a better fix than this.